src/reactjs/useSyncExternalStore.ts

```diff
@@ -0,0 +1,4 @@
+// @ts-ignore
+// eslint-disable-next-line import/no-unresolved
+import { useSyncExternalStore } from 'use-sync-external-store/shim/index.js'
```

Comment:
why would that import not resolve? It's the one that exists, right?

Reply:
fixed